src/core/i18n/translation/locales/en.ts

```diff
@@ -577,10 +577,12 @@ export const translation: ITranslations = {
             selectStakingPool: 'Selecting staking pool',
             unselectStakingPool: 'Unselecting staking pool',
             allBalanceNotice:
-                'Notice: Moonlet keeps some tokens in your account for future transaction fees.',
+                'Notice: Moonlet keeps a minimum of 100 ZIL in your account for future transactions like Stake, Claim, Unstake or Switch Node.',
```

Comment:
this message is shown on all blockchains, so with this change we will diplay 100 ZIL on Near also. If we want this we need to use parameters in transalation